system/jlib/jdebug.cpp

```diff
@@ -2030,7 +2030,7 @@ class CExtendedStats  // Disk network and cpu stats
                                      ((minnum&0x3F)==0) )
                                 {
                                     unsigned mm = (majnum<<16)+minnum;
-                                    majmin.appendUniq(mm);
+                                    disk_major_minor.appendUniq(mm);
```

Comment:
just curious, does use of appendUniq suggest could be duplicate entries?

Reply:
I don't think so, I was just being safe.